caas/Dockerfile

```diff
@@ -34,4 +37,15 @@ COPY jujuc /opt/
 COPY containeragent /opt/
 COPY pebble /opt/
 
+RUN chown -R juju /var/lib/juju
+RUN chown juju /opt/jujud
```

Comment:
I think we still want root to own the binaries but they should have the right octal for juju user to run them.